cli/src/pcluster/cli/exceptions.py

```diff
@@ -13,10 +13,9 @@
 # limitations under the License.
 
 import json
-from abc import ABC
 
 
-class CLIException(Exception, ABC):
+class CLIException(Exception):
```

Comment:
General comment: when fixing linters errors it's worth to add in the commit message the detail (e.g. error-code) in this way the reviewer can check the suggestion and understand why we're changing something.

In this way we'll also have the reason of the change saved in the git story of the modified line.